images/sponsors/Braid.svg

```diff
@@ -5,117 +5,72 @@
-         class="cls-2" />
-      <path
-         id="path239"
-         d="M 361.23,153.43 A 100.72,100.72 0 0 0 287.74,323 a 80.57,80.57 0 0 1 147,0 100.72,100.72 0 0 0 -73.5,-169.6 z"
-         class="cls-3" />
-      <path
-         id="path241"
-         d="M 361.23,354.89 A 100.44,100.44 0 0 1 287.74,323 a 80.59,80.59 0 1 0 147,0 100.48,100.48 0 0 1 -73.51,31.89 z"
-         class="cls-3" />
-      <g
-         id="g255"
-         class="cls-4">
-        <path
-           id="path243"
-           d="m 256.42,482.26 c 8.13,-3.71 13.27,-11.27 13.27,-21.26 0,-13 -8.71,-25.83 -35.11,-25.83 h -54.22 v 101.6 h 55.22 c 26.69,0 38.24,-12 38.24,-30 a 26.5,26.5 0 0 0 -17.4,-24.51 z M 212.61,460 h 15.84 c 5.28,0 8,2.57 8,7.14 0,4.57 -2.71,7.13 -8,7.13 h -15.84 z m 19.83,51.94 h -19.83 v -14.27 h 19.83 c 5.28,0 8,2.57 8,7.14 0,4.57 -2.72,7.13 -8,7.13 z"
-           class="cls-5" />
-        <path
-           id="path245"
-           d="m 308.64,477.7 v -14.56 h -29.1 v 73.63 h 29.68 v -28.25 c 0,-13.7 6.13,-19.41 17.69,-19.41 h 6.85 V 462.43 A 28.84,28.84 0 0 0 328.48,462 c -10.7,0 -17.27,5.28 -19.84,15.7 z"
-           class="cls-5" />
-        <path
-           id="path247"
-           d="m 416,485 c 0,-15 -11,-23.54 -37.53,-23.54 -27.25,0 -38.24,9.27 -38.95,27.25 v 0.43 h 29.68 v -0.43 c 0.43,-5.56 3.14,-7.27 8.84,-7.27 5.43,0 8.28,1.57 8.28,5.42 0,3.85 -2.85,5 -8.85,5.56 L 364,493.68 c -20.26,2 -28.11,10 -28.11,22.83 0,15.55 11.56,22.69 27.11,22.69 11.7,0 19.69,-4.14 23.69,-11 a 22.18,22.18 0 0 0 2.14,8.56 h 29.82 a 26,26 0 0 1 -2.71,-11.7 z m -29.68,23.69 c 0,8 -4,12 -11.56,12 -5.71,0 -8.7,-2.14 -8.7,-6.57 0,-3.85 2.28,-5.85 7,-6.42 l 6.85,-0.86 c 3.28,-0.42 5.28,-1.28 6.42,-3 z"
-           class="cls-5" />
-        <rect
-           id="rect249"
-           height="73.629997"
-           width="29.68"
-           y="463.14001"
-           x="423.51001"
-           class="cls-5" />
-        <path
-           id="path251"
-           d="m 512.42,435.17 v 37.53 c -4.42,-6.85 -11.41,-11.27 -21.83,-11.27 -19.55,0 -31.53,15.27 -31.53,38.53 0,23.26 12,38.52 31.53,38.52 11.13,0 18.41,-5 22.69,-12.7 v 11 H 542.1 V 435.17 Z m -11.13,80.48 c -8.13,0 -12.55,-5.28 -12.55,-15.55 0,-10.27 4.42,-15.7 12.55,-15.7 8.13,0 12.42,5.42 12.42,15.7 0,10.28 -4.28,15.55 -12.42,15.55 z"
-           class="cls-5" />
-        <path
-           id="path253"
-           d="m 438.35,428.2 a 14.94,14.94 0 1 0 14.94,14.93 14.93,14.93 0 0 0 -14.94,-14.93 z"
-           class="cls-5" />
-      </g>
-    </g>
-  </g>
+     id="title1350">Artboard 1</title>
+  <path
+     style="stroke-width:0.103689;fill:#231f20;fill-opacity:0.44736844"
```

Comment:
Could you please change the fill colors to `#666666`, and set the opacity to 100%? I like to do this in Inkscape by selecting everything and then typing "#666666ff" into the fill dialog (the `ff` at the end equals 255 opacity in hex).

Reply:
Thanks, @chromakode! Just pushed a new version. Let me know if it looks OK.